src/driver.h

```diff
@@ -1,4 +1,4 @@
-#include "drivers/inc/test.h"
+#include "drivers/inc/stm32f4xx.h"
```

Comment:
This was there as a hack. This line should be removed and instead people should include their driver's header before including CANlib. Ultimately the board `.h` file will have that include.